_news/20231103-tskit_arg_visualizer-0.0.1.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: news
+title: New software for visualising ARGs and tree sequences
+description: |
+  James Kitchens, a graduate student in the Coop lab in UC Davis, has
+  released the first version of his D3-based javascript
+  software for visualising Ancestral Recombination Graphs.
+  Interactive plots of ARGs in tskit format can be created
+  and displayed on web pages or in Jupyter notebooks.
+image: ../assets/images/news/20231103-tskit_arg_visualizer-0.0.1/viz.png
+timestamp: 2023-11-05
+author: Yan Wong
+---
+
+## New software for visualising ARGs and tree sequences
+
+![James Kitchens]({{ '/assets/images/news/20231103-tskit_arg_visualizer-0.0.1/james.jpg' |  relative_url}}){:class="news-float-left" style="width: 8em"}
+
+The standard visualization of a tree sequence is as a set of
+adjacent trees. However, tree sequences can also be though of as
+mathematical graphs. A new [D3](https://d3js.org)-based project called
```

Comment:
The phrase "mathematical graph" makes me wince - why not a mathematical tree?

Reply:
For a news item, I think it is justified. Most people (including me!) instinctively think of a graph as e.g. a scatterplot, not a vertex+edge structure. Do suggest another adjective though.

BTW, I agree that in specific graph-theory contexts, it's clear. But in a non-technical explanation, it's not. I'm happy with other suggestions to reduce ambiguity.

Reply:
Perhaps "topological graph" works better for you?

As an aside, here's the definition from e.g. Cambridge dictionaries - it doesn't even mention topology:

https://dictionary.cambridge.org/dictionary/english/graph

Reply:
If someone is that confused, I think they could equally think of "mathematical graph" as a line plot. That's just as mathematical as a set of nodes and edges. I just think there's no actual information imparted by adding "mathematical" in front of "graph".

This sentence doesn't really clarify much:

>  However, tree sequences can also be though of as mathematical graphs.

How about saying something like:

> However, the set of correlated trees in a tree sequence can also be thought of as a single graph structure.

Reply:
I changed it to "topological graph", but your phrasing works fine too.

I take your point about "mathematical" being also applicable for line plots.

Reply:
Just for my reference, I've just learned that a "topological graph" is a slightly different thing (an embedding of a graph in topological space). So "topological" is no good as an adjective.  "Graph network" is the best I can do, although it sounds a bit tautological.

Reply:
I think "network or graph" is the best you can do.

Reply:
I guess "directed graph(s)" might help to be more specific too.